### PR TITLE
Migrate files from TulipaEnergyModel.jl

### PR DESCRIPTION
**Pull request by gnawin**
~~Docs are not added yet. I tried to build the docs locally, but it says csv files not found, maybe it is because the files are not merged yet @datejada @abelsiqueira? So I want to merge these first.~~

Now documentation is also added.

### CODE REVIEW

**Comment by abelsiqueira** [comment]
I replied to your related comment on the issue, but I think I can reply here as well with something similar. I think we shouldn't depend on the files explicitly. One approach would be to include the files using the installation path of Tulipa:

```
using Pkg
dir = joinpath(Pkg.dir("TulipaEnergyModel"), "test/inputs/Norse")
```

Another would be to use only fake data, creating the `graph` object directly.

I suggested only the second approach in my comment, but now that I think about it, the first might be easier to maintain and update.

**Comment by gnawin** [comment]
Thanks! The first one seems to be a good one, I did not think of getting the inputs from the package directly. The path/dir is fully clear to me locally, but I'm confused/concerned where the documentation gets the inputs, e.g., when I showcase the plots in the documentation (like what we do here https://tulipaenergy.github.io/TulipaEnergyModel.jl/stable/tutorial/#Run-scenario), where does the code execute, should we do anything special then?

**Comment by gnawin** [comment]
@abelsiqueira I made the changes based your suggestion

```
using Pkg
dir = joinpath(Pkg.dir("TulipaEnergyModel"), "test/inputs/Norse")
```

My local tests are passing, some of the tests online are passing, but not for Julia nightly online. The error messages do not make sense to me. Do you have any hints what may go wrong? Btw, I do get an function depreciation warning for the above code when I run the tests locally.

**Comment by abelsiqueira** [comment]
Probably the nightly fails are because of the deprecation of `Pkg.,dir` - since it is deprecated now, it means that it will stop working on the next release.
You can change the code to

```Julia
import TulipaEnergyModel # Assuming that you haven't did this or the similar using
dir = joinpath(dirname(Pathof(TulipaEnergyModel)), "../test/inputs/Tiny")
```

Now, hopefully that also works on Julia 1.6.

**Comment by gnawin** [comment]
@abelsiqueira I updated accordingly, no warnings anymore locally, but the nightly still does not pass. 

image

It is a bit tricy to test because there is no warnings locally. Any ideas?

**Comment by abelsiqueira** [comment]
I think we can remove the tests in nightly, since this package won't see a lot of development, and it is secondary. Just comment out the lines in the workflow Test.yml that say nightly.

File: test/runtests.jl [hunk 1]
```diff
@@ -1,6 +1,10 @@
+using TulipaEnergyModel
 using TulipaPlots
 using Test
+using Pkg
```
Comment (abelsiqueira) [inline test/runtests.jl:4]: Don't need this anymore

File: test/Project.toml [hunk 2]
```diff
@@ -1,2 +1,4 @@
 [deps]
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```
Comment (abelsiqueira) [inline test/Project.toml:2]: Don't need this anymore

**Review by abelsiqueira — APPROVED** [review]
The Pkg needs to be removed. Otherwise, LGTM, thanks

File: test/runtests.jl (hunk 1 above)
Comment (gnawin) [reply test/runtests.jl]: Thank you. I'll fix this.

File: test/Project.toml (hunk 2 above)
Comment (gnawin) [reply test/Project.toml]: Thank you. I'll fix this.

**Pull request merged**